LogMonitor/src/LogMonitor/Main.cpp

```diff
@@ -77,179 +77,122 @@ void PrintUsage()
     wprintf(L"\tfile.\n\n");
 }
 
-
-bool StartMonitors(_In_ const PWCHAR ConfigFileName)
+void StartMonitors(_Out_ LoggerSettings& settings)

```

Comment:
why is this marked as output rather than input?

Reply:
I have changed the annotation to _In_ since it's an input to the function.

For some reason this works wit either _Out_ or _In_

Reply:
The annotations are purely for readability and have no impact on how it works